sdks/python/src/opik/integrations/openai/opik_tracker.py

```diff
@@ -32,6 +32,9 @@ def track_openai(
 
     decorator_factory = openai_decorator.OpenaiTrackDecorator()
 
+    if str(openai_client.base_url) != "https://api.openai.com/v1/":
```

Comment:
I think we should check if the host part is equal to `https://api.openai.com/`.
Otherwise I can think of scenarios when they for example change `v1`->`v2` and our logic stops working (it's way less likely to happen to host part).